src/pymorize/global_attributes.py

```diff
@@ -262,14 +14,7 @@ def set_global_attributes(ds: xr.DataArray, rule):
     """
     # Get the global attributes set on rule
     rule_attrs = rule.global_attributes_set_on_rule()
-
-    # Compute the other global attributes using the controlled vocabularies
-    global_attributes_factory = create_factory(GlobalAttributes)
-    GlobalAttributesClass = global_attributes_factory.get("CMIP6")
-    ga = GlobalAttributesClass(rule.controlled_vocabularies)
-    global_attributes = ga.get_global_attributes(rule_attrs)
-
+    global_attributes = rule.controlledvocabularies.global_attributes(rule_attrs)
```

Comment:
Are you sure it's `rule.controlledvocabularies` and not `rule.controlled_vocabularies`?